nautobot/dcim/models/device_components.py

```diff
@@ -604,18 +604,13 @@ def clean(self):
             # An interface cannot be its own parent
             if self.parent_interface_id == self.pk:
                 raise ValidationError({"parent_interface": "An interface cannot be its own parent."})
+
```

Comment:
```suggestion
```

Reply:
Was trying to make this function more self-consistent formatting - all the other "if" blocks have a space between them.